styles.css

```diff
@@ -0,0 +1,441 @@
+
+/* Задаем основные параметры страници */
+.body {
+  margin: 0 auto;
+  font-family: 'Montserrat', sans-serif;
+  max-width: 1440px;
+}
+
+/* box-sizing позволяет задавать свойства
+width и height для блока, а не его содержимого, */
+* {
+  box-sizing: border-box;
+}
+
+/* Основные пораметры для header */
+.header {
+  margin: 24px 60px 0px 37px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 15px;
+  line-height: 18px;
+}
+.header__logo {
+  width: 166px;
+  height: 51px;
+}
+.header__nav {
+  margin-left: 46px;
+  display: flex;
+  justify-content: space-around;
+  flex-wrap: wrap;
+  align-items: center;
+  width: 40%
+}
+.header__nav-link {
+  display: block;
+  text-decoration: none;
+  color: #000;
+  line-height: 32px;
+}
+/* меняем поведение ссылки при наведении */
+.header__nav-link:hover {
+  color: #e51937;
+  border-bottom: 2px solid;
+  border-radius: 2px;
+  transition: color .2s;
+}
+.header__menu {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 204px;
+}
+.header__menu-burger {
+   fill: #e51937;
+   width: 24px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/max_osipov/pullRequest?prid=5575857)